openlogin/homepage/index.lib.ts

```diff
@@ -1,10 +1,43 @@
 import * as playwright from "@playwright/test";
 import { env_map } from "../../utils/index";
+import * as dotenv from "dotenv";
+dotenv.config();
 
 export interface TestArgs {
   openloginURL: string;
+  user: {
```

Comment:
please remove any reference to user, its not being used anywhere.

Reply:
done